assets/alias.yaml

```diff
@@ -7,19 +7,8 @@ acoustic_oos:
   - background_noise
   - background_speech
   - other_language
-oos:
+lexical_oos:
```

Comment:
`alias.yaml` captures all the aliasing that needs to be happen before evaluation. each new alias is associated with a list of existing intents. each intent in this list is being aliased, because we want to treat them as the same during evaluation.

for eg. an SLU model could be predicting a separate class for each oos intent, but we still want to treat them as a single OOS class (in the default `alias.yaml` this example is replaced by its equivalent in two OOS classes)